package/gluon-geolocator/files/usr/lib/micron.d/geolocator

```diff
@@ -0,0 +1 @@
+*/5 * * * *     sh /lib/gluon/geolocator/geolocator.sh;
```

Comment:
* Unneded semicolon
* Remove the `sh`, the script is executable and has a shebang line